src/components/client/product/product.tsx

```diff
@@ -0,0 +1,130 @@
+            >
+              Новинка
+            </CardItem>
+          )}
+          {product.statusLabel?.includes('sale') && (
+            <CardItem
+              translateZ={30}
+              className="icon-sale relative z-10 flex w-fit items-center rounded-sm bg-[#A90909] px-2 py-[2px] uppercase"
+            >
+              Акція
+            </CardItem>
+          )}
+        </div>
+        <CardItem
+          translateZ={70}
+          className="z-2 relative grid max-w-[229px] grow-0 place-content-center px-2 py-8"
+        >
+          <Image
+            src={product.img ?? ''}
+            alt={productName}
+            width={229}
+            height={229}
+            className="relative aspect-square h-full w-full object-contain"
+          />
+        </CardItem>
+        <div>Visited: {product.visited}</div>
+        <div className="flex w-full items-center justify-between gap-2">
+          <CardItem translateZ={50}>
+            <h3 className="text-xs font-medium uppercase sm:text-sm md:text-base lg:text-lg xl:text-xl">
+              {productName}
+            </h3>
+          </CardItem>
+          <CardItem
+            translateZ={20}
+            className="sm:text-md flex w-[80px] shrink-0 items-center text-nowrap text-base font-medium sm:w-[90px] md:w-[100px] md:text-lg xl:text-xl"
+          >
+            <Select
+              onValueChange={(value) => setSelectedWeightIndex(Number(value))}
+            >
+              <SelectTrigger>
+                <SelectValue
+                  placeholder={`${product.variables[selectedWeightIndex].weight} г`}
+                />
+              </SelectTrigger>
+              <SelectContent>
+                {product.variables.map((weight, index) => (
+                  <SelectItem key={index} value={index.toString()}>
+                    {weight.weight} г
+                  </SelectItem>
+                ))}
+              </SelectContent>
+            </Select>
+          </CardItem>
+        </div>
+        <div className="flex w-full items-end justify-between gap-2">
+          <CardItem translateZ={20}>
+            {selectedNewPrice ? (
+              <>
+                {selectedPrice && (
+                  <p className="text-xs font-semibold text-[#7D7D7D] line-through sm:text-sm md:text-base lg:text-lg xl:text-lg">
+                    {selectedPrice} грн
+                  </p>
+                )}
+                <p className="text-sm font-semibold sm:text-base lg:text-lg xl:text-xl">
+                  {selectedNewPrice} грн
+                </p>
+              </>
+            ) : (
+              selectedPrice && (
+                <p className="text-sm font-semibold sm:text-base lg:text-lg xl:text-xl">
+                  {selectedPrice} грн
+                </p>
+              )
+            )}
+          </CardItem>
+          <CardItem translateZ={40}>
+            <Button type="button" variant="button">
```

Comment:
create function who push selected product to console log,     id
name
count
price
newPrise

Reply:
Created